free-2.md

```diff
@@ -34,36 +38,44 @@ _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+I imagine the program can be run on the terminal. At first, the user will be able to import a picture they already own, or input keywords to get a picture randomly from Google.
```

Comment:
Can you elaborate on what type of input keywords would be available to the user? What will the keywords do or are they just intended as image search keywords?

I think it would be neat to specify all sorts of image manipulations like number of colors, thickness of lines, skews, merges, and patterns.